fabfile.py

```diff
@@ -13,13 +13,41 @@
 CURRENT_DIR = os.path.dirname(__file__)
 
 
+TIERS = {
+    'production': {
+        'hosts': ['root@104.131.92.59'],
+    },
+    'staging': {
+        'hosts': ['root@162.243.226.226'],
+    },
+}
+
+
+def tier_set(tier_name='staging'):
+    env.tier = tier_name
+    for option, value in TIERS[env.tier].items():
+        setattr(env, option, value)
+
+
+@task
+def production():
+    tier_set('production')
+
+
+@task
+def staging():
+    tier_set('staging')
+
+
 @task
 def install_salt():
+    require('tier', provided_by=(staging,production))
```

Comment:
This line (and similar) should have a space after the comma in `staging,production`.

Also, this... doesn't seem to do anything? i.e. if I remove 'staging' from the list for deploy, I can still run `fab staging deploy`...